uploader.html

```diff
@@ -178,96 +175,88 @@ <h4 class="modal-title">Upload <%= filetype %></h4>
+          }
+        ];
+        var mediaType = mediaTypes[0][filetype];
 
-      // init
-      self.init = function () {
         $(document).ready(function() {
-          createModalHeader();
+          createModalHeader(mediaType['text']);
+        });
+        $(document).on('click', opts.selectFileSelector, function() {
+          _private.createSelectFileSection(mediaType['text']);
         });
-        $(document).on('click', opts.selectFileSelector, _private.createSelectFileSection);
         $(document).on('change', opts.fileFieldSelector, function(event) {
-          _private.createSelectionRow();
+          _private.createSelectionRow(mediaType['text']);
           _private.collectFileData(event);
         });
-        // $(document).on('click', opts.submitFileSelector, function() {
-        //   _private.createSelectionRow();
-        // });
-        $(document).on('click', opts.fullSizeImage, _private.showFullSize);
       };
 
       // public functions
-      var createModalHeader = function() {
+      var createModalHeader = function(filetype) {
         var selectHeader = $('#js-create-header');
         var selectHeaderTemplate = _.template(selectHeader.text());
-        console.log(selectHeaderTemplate);
         $(selectHeader).html(selectHeaderTemplate(
-          {filetype: "image"}
+          {filetype: filetype}
         ));
-        console.log('Create Modal Header function working!');
       }
 
       // private functions
       // create select file section
-      _private.createSelectFileSection = function() {
+      _private.createSelectFileSection = function(filetype) {
         var selectFileRow = $('#js-select-file-section');
         var fileRow = $('#select-file-section');
         var selectFileTemplate = _.template(selectFileRow.text());
         $(fileRow).html(selectFileTemplate(
-          {filetype: "image"}
+          {filetype: filetype}
         ));
-        console.log('Create File Section function working!');
       }  
   
       // create selection row
-      _private.createSelectionRow = function() {
+      _private.createSelectionRow = function(filetype) {
         var selectionRow = $('#js-selected-media-section');
         var mediaRow = $('#selected-media-section');
         var selectionTemplate = _.template(selectionRow.text());
         $(mediaRow)
           .addClass("spacing")
           .html(selectionTemplate(
-            {filetype: "image"}
+            {filetype: filetype}
             ));
-        console.log('Scoped Selection Section function working!');
       }
 
       // get data from file selector
       _private.collectFileData = function(event) {
+        var image = {};
```

Comment:
curious about why this was moved out here. The previous way seems better since (as far as I can see) it's only referenced inside the `onload` handler.

Reply:
I was hoping to use it elsewhere, but removed that code because it didn't work and muddied the testing.

Reply:
@millerdev Can you take a look at this?  Hoping to get this PR closed this week :)